README.md

```diff
@@ -233,28 +233,30 @@ You can also use key paths (for both dictionary keys and array indexes) to unbox
 
 ```json
 {
-    "name": "John",
-    "age": 27,
-    "activities": {
-        "running": {
-            "distance": 300
-        }
-    },
-    "devices": [
-        "Macbook Pro",
-        "iPhone",
-        "iPad"
-    ]
+	"name": "John",
```

Comment:
I'm sorry about that—I had opened the README in iA Writer (my markdown editor of choice) and the tabbing was different and it just got messed up. I didn't intend to change that.